File: Primeiro periodo/AED-1/CSharp/Trabalho Prático 01/Hipotenusa.cs
using System;

public class Hipotenusa {
    public static void Main(){
        Console.Write("O seguinte programa calcula a hipotenusa ");
        Console.WriteLine("de um triângulo retângulo.");
        Console.Write("Digite o primeiro cateto: ");
        double catetoA = Convert.ToDouble(Console.ReadLine());
        Console.Write("Digite o segundo cateto: ");
        double catetoB = Convert.ToDouble(Console.ReadLine());
        double hipotenusa = Math.Sqrt((Math.Pow(catetoA, 2) + (Math.Pow(catetoB, 2))));
        Console.WriteLine($"O valor da hipotenusa é: {hipotenusa}");
    }
}

File: Primeiro periodo/AED-1/Java/Trabalho Prático 01 - AED1/Hipotenusa.java
// Autor: Luiz Junio <luisjuniorbr@gmail.com>

import java.util.Scanner;

class Hipotenusa {
    public static void main (String [] args)
    {
        Scanner s = new Scanner (System.in);
        System.out.println("O seguinte programa calcula a hipotenusa de um triângulo retângulo.");
        System.out.print("Digite o primeiro cateto: ");
        double cateto_a = s.nextDouble();
        System.out.print("Digite o segundo cateto: ");
        double cateto_b = s.nextDouble();
        double hipotenusa = Math.sqrt((Math.pow(cateto_a,2) + Math.pow(cateto_b,2)));
        System.out.print("O valor da hipotenusa é: " + hipotenusa);
    }
}
